.circleci/config.yml

```diff
@@ -76,17 +76,17 @@ jobs:
           command: |
             sudo apt-get update && sudo apt-get install make
       - run:
-          name: Setup
```

Comment:
Install clusterctl so we can deploy CAPI components. We need them so we can have the conversion webhooks and have a more realistic test cluster.